cores.json

```diff
@@ -0,0 +1,466 @@
+[
+    {
+        "name": "mame2003-libretro",
+        "makeoptions": {
+            "buildpath": "./",
+            "makescript": "Makefile",
+            "arguments": []
+        },
+        "options": {
+            "file": "MAME 2003 (0.78)/MAME 2003 (0.78).opt",
+            "settings": {
+                "mame2003_skip_disclaimer": "enabled",
+                "mame2003_skip_warnings": "enabled"
+            }
+        },
+        "license": "https://github.com/EmulatorJS/mame2003-libretro/blob/master/LICENSE.md",
+        "repo": "https://github.com/EmulatorJS/mame2003-libretro"
+    },
+    {
+        "name": "vice_x64",
+        "makeoptions": {
+            "buildpath": "./",
+            "makescript": "Makefile",
+            "arguments": [
+                "EMUTYPE=x64",
+                "SECONDARG=Poo"
```

Comment:
Was this a test?

Reply:
OH MY... :D Yeah, was testing that I could pass multiple arguments.

Cleaned up!